charge_lnd/strategy.py

```diff
@@ -88,7 +96,8 @@ def strategy_ignore(channel, policy, **kwargs):
         time_lock_delta=KEEP,
         inbound_base_fee_msat=KEEP,
         inbound_fee_ppm=KEEP,
-        disabled=KEEP
+        disabled=KEEP,
+        circuitbreaker_params=KEEP
```

Comment:
strategy_ignore is called when disabling a channel. We should overwrite ciricuitbreaker_params to DONTCARE in this case.